pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -228,30 +236,34 @@ def on_epoch_end(self, epoch, logs=None):
         self.epochs_since_last_save += 1
         if self.epochs_since_last_save >= self.period:
             self.epochs_since_last_save = 0
-            filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
-            if self.save_best_only:
+            if self.save_all or self.save_last:
+                overwrite = not self.save_all
+                filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
```

Comment:
consider rename `self.filepath` because in this context it is a folder path

Reply:
Ok, I was following the existing naming scheme but you are right